### ISSUE TITLE
Data for `active` in Hubei, China on 03.24 is unrealistically high

### ISSUE THREAD

**Issue by zbraniecki** [lazd/coronadatascraper#357]
There's a sudden jump in `active` likely due to lack of `recovered`:

```json
      "2020-3-23": {
        "cases": 67800,
        "deaths": 3153,
        "recovered": 59879,
        "active": 4768,
        "growthFactor": 1
      },
      "2020-3-24": {
        "cases": 67801,
        "deaths": 3160,
        "active": 64641,
        "growthFactor": 1.0000147492625369
      }
```

**Comment by lazd** [comment]
This is 100% because JHU stopped providing `recovered`.

The only solution is to remove `active` -- we're not getting `recovered` data consistently from anywhere, so it's starting to become meaningless.

@qgolsteyn @camjc @chunder thoughts?

**Comment by chunder** [comment]
We should omit `active` if `recovered` is null, right? We should not treat `recovered == null` as if it is 0.

**Comment by chunder** [comment]
This will naturally cause `active` to disappear from ~most locations.

**Comment by qgolsteyn** [comment]
I think active should be kept, but set to undefined if we are missing data for `cases`, `deaths`, or `recovered`.

See #354 for another `active` related bug

**Comment by qgolsteyn** [comment]
Closed with 84e8a277
